Expensify/App
# Fix report icon logic for newly created policy rooms

@deetergp, please review when you get the chance.
CC: @neil-marcellini for help in testing this out on iOS/android and providing screenshots!

### Details
Fixes bug that causes newly created policy rooms to crash when you click on their details. Since the rooms have 

### Fixed Issues

$ https://github.com/Expensify/App/issues/7182
$ https://github.com/Expensify/App/issues/7055
$ https://github.com/Expensify/App/issues/7262

### Tests/ QA Steps
1. Log into NewDot with an account that has a workspace or create a workspace using the plus on the bottom left. Make sure you are the only person in the workspace.
2. Using the same plus on the bottom left, choose `New Room`. Fill in the options in the modal and after the new room is created, verify that you see the couch icon for the room after it is created:
![image](https://user-images.githubusercontent.com/19364431/150612421-a7c376f6-2ce6-4920-825c-c9a2b74e744a.png)
3. Click on that icon and verify you can access the room details like in the screenshots below. The room's settings should also be accessible.



### Tested On


### Screenshots


#### Web/ Mobile Web
![image](https://user-images.githubusercontent.com/19364431/150612461-e2f6f289-f389-4128-975c-9344f089809d.png)
![image](https://user-images.githubusercontent.com/19364431/150612583-29319159-e9f3-4d62-8a71-a756369024f5.png)


#### iOS
![image](https://user-images.githubusercontent.com/19364431/150612501-769f9670-eddf-4a2a-9d8c-09d7710969af.png)


#### Android
![image](https://user-images.githubusercontent.com/19364431/150612514-475a09ae-aacb-4a6a-821e-733029955b87.png)
